src/python/BFinitPassthrough.py

```diff
@@ -84,4 +84,7 @@ def bf_passthrough_init(port, requestedBaudrate, half_duplex=False):
     except:
         requestedBaudrate = 420000
     port = serials_find.get_serial_port()
-    bf_passthrough_init(port, requestedBaudrate)
+    try:
+        bf_passthrough_init(port, requestedBaudrate)
```

Comment:
Why this is needed? 
this "main" function is not called during normal upload process :)

Reply:
Yes, it is. In the `DIY_*_RX_*_via_UART` targets.

Reply:
oh, ESP stuff...
I have skipped those since those targets are not working :)